packages/components/src/KaizenProvider/_docs/KaizenProvider.mdx

```diff
@@ -6,7 +6,7 @@ import migrationFETToV1 from "!!raw-loader!./code-diffs/migration-fe-template-to
 import migrationFETBonus from "!!raw-loader!./code-diffs/migration-fe-template-bonus.diff"
 import { KaioNotification } from "../../../../../storybook/components"
 
-<Meta title="Components/🛠️ Kaizen Provider/Installation" />
+<Meta title="Components/Kaizen Provider/Installation" />
```

Comment:
The emoji is something CK added to keep it at the top

Reply:
Yeah fair. It makes linking to the Kaizen Provider stories difficult though. I've moved it to the top again by adding it to SB's sort order instead.